.travis.yml

```diff
@@ -29,11 +30,15 @@ after_failure:
 
 after_success:
         - ./rebar3 coveralls send
-        - if [ $PRESET = 'internal_mnesia' ] && [ $TRAVIS_OTP_RELEASE = "19.3" ]; then tools/travis-build-and-push-docker.sh; fi
+        - if [ -n "${DOCKERHUB_PASS}" ] && [ $PRESET = 'internal_mnesia' ] && [ $TRAVIS_OTP_RELEASE = "19.3" ];
+            then tools/travis-build-and-push-docker.sh;
+          fi
 
 after_script:
-        - sudo pip install awscli --ignore-installed six
-        - if [ $TRAVIS_SECURE_ENV_VARS == 'true' ]; then tools/travis-upload-to-s3.sh; fi
+        - tools/travis-prepare-log-dir.sh
+        - if [ -n "${AWS_SECRET_ACCESS_KEY}" ]; then tools/travis-upload-to-s3.sh; fi
+        - if [ -n "${GDRIVE_SERVICE_ACCOUNT_CREDENTIALS}" ]; then tools/travis-upload-to-gdrive.sh; fi
+        - dd if=/dev/urandom of=/tmp/serviceAccountCredentials bs=1 conv=notrunc count=$(wc -c /tmp/serviceAccountCredentials | awk '{print $1}')
```

Comment:
short description what it does.

Reply:
It's a sad day when a simple `dd` has to be commented for programmers :(